msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -995,6 +995,46 @@ The `extensions.alternates` property is used to hide or prioritize specific in-m
 |`hide.customOfficeAddin`| | | | Configures how to hide an in-market add-in that isn't distributed through AppSource.|
 |`hide.customOfficeAddin.officeAddinId`|String | 64 characters | ✔️ | Specifies the ID of the in-market add-in to hide. The GUID is taken from the app manifest `id` property if the in-market add-in uses the JSON app manifest. The GUID is taken from the `<Id>` element if the in-market add-in uses the XML app manifest. |
 
+## actions
+
+The object is an array of action objects. This block is required only for solutions that provides Actions.
+
+|Name| Type| Maximum size | Required | Description|
+|---|---|---|---|---|
+|`id`| String | 64 characters | ✔️ | Specifies the ID for the action, which is passed to the code file. |
```

Comment:
This description is misleading and missing information, please use the original one I proposed:

An identifier string in the default locale that is used to catalog actions. Must be unique across all actions for this app. Example: “openDocInContoso” 